app/assets/stylesheets/widgets/home_page/blog_stories.scss

```diff
@@ -41,7 +41,7 @@
       }
       .title {
         text-transform: none;
-        color: #98002e;
+        color: #a50034;
```

Comment:
You should be able to use the SCSS variable for all these stylesheets. Is there a reason why they're hardcoded instead of using `$cu-red`? Even if it was previously hardcoded for whatever reason, you should be able to change it to the SCSS variable. If you test it and it doesn't work, let me know.